bundle.yaml

```diff
@@ -2,17 +2,21 @@
 # See LICENSE file for licensing details.
 
 bundle: kubernetes
+name: argo-workflows
 applications:
   argo-controller:
-    charm: cs:argo-controller
-    source: ./operators/argo-controller
+    charm: ch:argo-controller
+    channel: latest/edge
+    source: ./charms/argo-controller
```

Comment:
Does this solve the issue of juju not liking the channel being specified in the `bundle.yaml` file for local .charm files?

Reply:
I think the problem still exists.  I believe the issue is that because the bundle has a local source at all (eg: points to a local file) then it will never work in juju.  I'm not sure why there'd be a local source file specified in this bundle anyway though, I think juju bundle would apply that on the fly if needed